src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -14,7 +14,7 @@
  * Only support BAN requests on purpose, to be able to invalidate cache for a
```

Comment:
`BAN` not updated right?

Reply:
yeah, that was an old comment... Removed it